### PR TITLE
Test readme

### PR DESCRIPTION
**Pull request by tbruno**
In this branch readme was fixed.

### CODE REVIEW

**Comment by nanovazquez** [comment]
Don't forget to `squash` before commit

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 the-wire
 ========
+bla
```
Comment (nanovazquez) [inline README.md:3]: Use uppercase here

**Comment by tbruno** [comment]
Solved! Squashed

File: README.md (hunk 1 above)
Comment (tbruno) [inline README.md:3]: Its ok

**Comment by nanovazquez** [comment]
LGTM :+1:

**Pull request merged**